packages/showcases/core/csf/components/indicators/rating.csf.js

```diff
@@ -71,4 +71,10 @@ export const argTypes = {
     defaultValue: false,
     control: { type: 'boolean' },
   },
+  ariaLabel: {
+    type: { name: 'string', required: false },
```

Comment:
type object

Reply:
I think in this case the accessibility should be always required.

What do you think @lauthieb & @thibault-mahe ?

Reply:
Done for type